e2e/cypress/integration/messaging/post_textbox_height_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+
+        // # Add two posts
+        cy.postMessage('test post 1');
+        cy.postMessage('test post 2');
+    });
+
+    it('M18700 - Leave a long draft in reply input box', async () => {
+        // # Get latest post id
+        cy.getLastPostId().then((latestPostId) => {
+            // # Click reply icon
+            cy.clickPostCommentIcon(latestPostId);
+
+            // # Make sure that text box has initial height
+            getTextBox().should('have.css', 'height', '100px');
+
+            // // # Write a long text in text box
+            getTextBox().type('test\n\n\n\n\n\n');
```

Comment:
This is not getting posted in recent Cypress version.  Please change to,
```suggestion
            getTextBox().type('test{shift}{enter}{enter}{enter}{enter}{enter}{enter}');
```

Reply:
Done :+1: